packages/mui-material/src/Tooltip/Tooltip.js

```diff
@@ -118,9 +118,11 @@ const TooltipTooltip = styled('div', {
     ];
   },
 })(({ theme, ownerState }) => ({
-  backgroundColor: alpha(theme.palette.grey[700], 0.92),
-  borderRadius: theme.shape.borderRadius,
-  color: theme.palette.common.white,
+  backgroundColor: theme.vars
+    ? `rgba(${theme.vars.palette.grey.darkChannel} / 0.92)`
```

Comment:
This doesn't work, we need to have channel for that specific color. Currently we are not generating channels for the grey's colors, but we can update the `extendTheme` to do this. If you want to give this a try, take a look at https://github.com/mui/material-ui/blob/master/packages/mui-material/src/styles/experimental_extendTheme.js#L62

You will notice that currently this doesn't work, as the tooltip in light mode have white background (which is the same as the text color)

Reply:
Added the token https://github.com/mui/material-ui/pull/32594/files#diff-41f50b7e71d281679755abc4d49b2011dffdb5e08c883077a7d7d09a7e5d3425R60